tools/mksurfdata_esmf/src/mkagfirepkmonthMod.F90

```diff
@@ -21,8 +21,8 @@ module mkagfirepkmonthMod
   public  :: mkagfirepkmon       ! Set agricultural fire peak month
 
   integer , parameter :: min_valid = 1
-  integer , parameter :: max_valid = 12
-  integer , parameter :: unsetmon = 13
+  integer , parameter :: max_valid = 13
+  integer , parameter :: unsetmon = 0
```

Comment:
From meeting with @samsrabin 
1) unsetmon probably unnecessary now, plus error check below will fail with unsetmon = 0
2) add comments in this module to explain things
3) add unit test in PFunit (@samsrabin adding separate comment about this)